src/supervise-daemon/meson.build

```
@@ -1,5 +1,5 @@
 executable('supervise-daemon',
-  ['supervise-daemon.c', misc_c, plugin_c, schedules_c, usage_c, version_h],
+  ['supervise-daemon.c', '../start-stop-daemon/pipes.c', misc_c, plugin_c, schedules_c, usage_c, version_h],
```

I wonder if would make sense to move `../start-stop-daemon/pipes.c` to `shared/pipes.c`?
```suggestion
  ['supervise-daemon.c', pipes_c, misc_c, plugin_c, schedules_c, usage_c, version_h],
```

---

```diff
diff --git a/src/shared/meson.build b/src/shared/meson.build
index b80b242e..04c70fef 100644
--- a/src/shared/meson.build
+++ b/src/shared/meson.build
@@ -10,6 +10,10 @@ schedules_c = files([
   'schedules.c',
   ])
 
+pipes_c = files([
+  'pipes.c',
+  ])
+
 if selinux_dep.found()
   selinux_c = files([
     'selinux.c',
diff --git a/src/start-stop-daemon/pipes.c b/src/shared/pipes.c
similarity index 100%
rename from src/start-stop-daemon/pipes.c
rename to src/shared/pipes.c
diff --git a/src/start-stop-daemon/pipes.h b/src/shared/pipes.h
similarity index 100%
rename from src/start-stop-daemon/pipes.h
rename to src/shared/pipes.h
diff --git a/src/start-stop-daemon/meson.build b/src/start-stop-daemon/meson.build
index d363ff94..6cae791d 100644
--- a/src/start-stop-daemon/meson.build
+++ b/src/start-stop-daemon/meson.build
@@ -1,5 +1,5 @@
 executable('start-stop-daemon',
-  ['start-stop-daemon.c', 'pipes.c', misc_c, schedules_c,
+  ['start-stop-daemon.c', pipes_c, misc_c, schedules_c,
        selinux_c, usage_c, version_h],
   c_args : [cc_audit_flags, cc_branding_flags, cc_pam_flags, cc_cap_flags, cc_selinux_flags],
   link_with: [libeinfo, librc],
```